playbooks/provisioner/beaker/main.yaml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Pre-provisioning tasks
+  hosts: localhost
+  sudo: no
+  tasks:
+    - name: Group by provisioner type
+      group_by: key={{ provisioner.type }}
+
+    - name: Add the host to the inventory
+      add_host:
+        name="host0"
+        groups="provisioned"
+        ansible_fqdn="{{ provisioner.fqdn }}"
+        ansible_ssh_host="{{ provisioner.fqdn }}"
+        ansible_ssh_user="{{ provisioner.ssh_user }}"
+        ansible_ssh_password="{{ provisioner.ssh_password }}"
```

Comment:
We just need to make sure the "provisioner.ssh_password" is set. Only thing I could find in configs was "provisioner.username", though.